recipes/speg/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "speg" %}
+{% set version = "0.2" %}
+{% set sha256 = "474b083021379a4e6aa1465879eeaeab3f70cbf474d52f9845f0872741aa3635" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - speg
+
+about:
+  home: https://github.com/avakar/speg
+  license: MIT
+  license_family: MIT
```

Comment:
Please ask upstream to add the license file to `MANIFEST.in`. Also please add the following here.

```yaml
  # Needs license file in the `sdist`.
  # Waiting on <issue/PR> to be addressed before uncommenting.
  # xref: <issue/PR link>
  # license_file: LICENSE